### DISCUSSION TITLE
[Suggestions] Dimension Door spell, effect of moving the hero on the adventure map

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#3602] [General]
In the original game, the hero is always centered relative to the adventure map display area when the dimension door spell is used to move, the adventure map is moved during the transition effect.

In fheroes2, the hero disappears from the adventure map display area, the adventure map is moved in the adventure map display area and the hero then appears.

**Edit 2021/04/09.**

In original game:

https://user-images.githubusercontent.com/43583994/114218773-25852d00-996a-11eb-805f-1ea40ecac77c.mp4

In fheroes2:

https://user-images.githubusercontent.com/43583994/114218863-451c5580-996a-11eb-921d-ff3051cf2c39.mp4

Here is the test scenario file:
[Test locations.zip](https://github.com/ihhub/fheroes2/files/6287412/Test.locations.zip)

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , I moved this issue to Discussions area for now. Yes, here's a certain difference in a way how Dimension door works. First of all, in the original game it's done due to limitation of the screen area while in our case a player usually has much bigger one.

@Branikolog , your thoughts?

**Reply by Branikolog** [reply]
I have no objections on the way it's working currently.

**Reply by LeHerosInconnu** [reply]
Hello @ihhub and @Branikolog,

In the original game, the effect gives the impression of using something "magical" when the hero moves, while in fheroes2, the hero just seems to jump from one place to another.

If it's a problem of hardware or software resources, would it be possible to reproduce the effect of the original game only for the lowest resolutions of fheroes2 (for example up to 1280 x 960 resolution)?
This would allow to keep the experience of the original game when it is possible.

**Comment by LeHerosInconnu** [comment]
Other videos with which the transition effect is more visible, and more recent save files (1.0.3 7084) for this.

In original game:

https://user-images.githubusercontent.com/43583994/236639637-593fdf79-4934-4f9f-b716-9e955b6b16cc.mp4

In fheroes2:

https://user-images.githubusercontent.com/43583994/236639647-a3e31aaa-b89a-4546-b9f7-d0bd4a53c7a9.mp4

Here are the test scenario and save files (1.0.3 7084):
[0 9988h and saves.zip](https://github.com/ihhub/fheroes2/files/11412919/0.9988h.and.saves.zip)
